google-guest-agent-manager.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Google Compute Engine Guest Agent Plugin Manager
+# Start after network is online.
+#   Debian/Ubuntu 16.04: networking.service
+#   SLES/EL7: network.service (SLES via wicked.service)
+#   EL8: NetworkManager.service
+#   COS/Ubuntu 18.04+: systemd-networkd.service
+After=network-online.target syslog.service
+After=network.service networking.service NetworkManager.service systemd-networkd.service
```

Comment:
can we have this run after guest-agent? That way Plugin Manager doesn't need to worry about Network not being ready case

Reply:
Yep, we can do it.

Reply:
done.